src/components/Create/components/pages/PayoutsPage/components/TreasuryOptionsRadio.tsx

```diff
@@ -0,0 +1,174 @@
+    switch (treasuryOption) {
+      case 'amount':
+        return t`A fixed amount of ETH can be paid out from your project each cycle. You can send specific ETH amounts (or ETH amounts based on USD values) to one or more recipients. Any remaining ETH will stay in your project for token redemptions or use in future cycles.`
+      case 'unlimited':
+        return t`All of your project's ETH can be paid out at any time. You can send percentages of that ETH to one or more recipients.`
+      case 'zero':
+        return t`None of your project's ETH can be paid out. All ETH will stay in your project for token redemptions or use in future cycles.`
+    }
+  }, [treasuryOption])
+
+  const switchToAmountsPayoutSelection = useCallback(
+    (newDistributionLimit: ReduxDistributionLimit) => {
+      setDistributionLimit(parseInt(fromWad(newDistributionLimit.amount)))
+      setCurrency(newDistributionLimit.currency)
+      setTreasuryOption('amount')
+      switchingToAmountsModal.close()
+    },
+    [setDistributionLimit, switchingToAmountsModal, setCurrency],
+  )
+
+  const switchToUnlimitedPayouts = useCallback(() => {
+    setDistributionLimit(undefined)
+    setTreasuryOption('unlimited')
+    switchingToUnlimitedModal.close()
+  }, [switchingToUnlimitedModal, setDistributionLimit])
+
+  const switchToZeroPayoutSelection = useCallback(() => {
+    setPayoutSplits([])
+    setDistributionLimit(0)
+    setTreasuryOption('zero')
+    switchingToZeroAmountsModal.close()
+  }, [setDistributionLimit, setPayoutSplits, switchingToZeroAmountsModal])
+
+  const onTreasuryOptionChange = useCallback(
+    (option: TreasurySelection) => {
+      const currentOption = treasuryOption
+      const payoutsCreated = payoutSplits.length
+      if (option === currentOption) return
+      if (option === 'amount' && payoutsCreated) {
+        switchingToAmountsModal.open()
+        return
+      } else if (option === 'amount' && !payoutsCreated) {
+        setDistributionLimit(0)
+      }
+
+      if (option === 'unlimited' && payoutsCreated) {
+        switchingToUnlimitedModal.open()
+        return
+      } else if (option === 'unlimited' && !payoutsCreated) {
+        switchToUnlimitedPayouts()
+      }
+
+      if (option === 'zero' && payoutsCreated) {
+        switchingToZeroAmountsModal.open()
+        return
+      } else if (option === 'zero' && !payoutsCreated) {
+        switchToZeroPayoutSelection()
+      }
+
+      setTreasuryOption(option)
+    },
+    [
+      treasuryOption,
+      payoutSplits.length,
+      switchingToAmountsModal,
+      switchingToUnlimitedModal,
+      setDistributionLimit,
+      switchingToZeroAmountsModal,
+      switchToZeroPayoutSelection,
+      switchToUnlimitedPayouts,
+    ],
+  )
+
+  useEffect(() => {
+    if (distributionLimit === undefined) {
+      setTreasuryOption('unlimited')
+    } else if (distributionLimit > 0) {
+      setTreasuryOption('amount')
+    }
+  }, [distributionLimit])
```

Comment:
Its slightly more verbose, but i think the following is more desirable: remove this useEffect in favor of calling `setTreasuryOption` explicitly where-ever `distributionLimit` is being set. 

This way the code is easier to follow/debug.

Reply:
The only reason for this is that I wanted the radio to automatically change when someone adds a payout starting from zero. 

The only way around it outside of this useEffect would be to set the treasuryOption redux parameter inside PayoutsTable. I would rather keep redux totally out of the payouts table

Reply:
eh. got it